Android build error in empty RN project: User has minSdkVersion 21 but library was built for 23 [//ReactAndroid/fabricjni]

## Environment
**OS**: Windows 10
**react-native**: 0.74.1 (latest)

## Problem
While trying to build the project for Android after installing the package in an empty React Native project, I get the following error:
```
> Configure project :react-native-quick-crypto
QuickCrypto: node_modules/ found at: <rootDir>\node_modules

> Task :react-native-quick-crypto:processDebugManifest
package="com.margelo.quickcrypto" found in source AndroidManifest.xml: <rootDir>\node_modules\react-native-quick-crypto\android\src\main\AndroidManifest.xml.
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported, and the value is ignored.
Recommendation: remove package="com.margelo.quickcrypto" from the source AndroidManifest.xml: <rootDir>\node_modules\react-native-quick-crypto\android\src\main\AndroidManifest.xml.

> Task :react-native-quick-crypto:configureCMakeDebug[arm64-v8a]
[CXX5304] Observed package id 'cmdline-tools;latest' in inconsistent location 'C:\Users\folivera\AppData\Local\Android\Sdk\cmdline-tools\old' (Expected 'C:\Users\folivera\AppData\Local\Android\Sdk\cmdline-tools\latest')
[CXX5304] Observed package id 'cmdline-tools;latest' in inconsistent location 'C:\Users\folivera\AppData\Local\Android\Sdk\cmdline-tools\old' (Expected 'C:\Users\folivera\AppData\Local\Android\Sdk\cmdline-tools\latest')
C/C++: <rootDir>\node_modules\react-native-quick-crypto\android\CMakeLists.txt debug|arm64-v8a : com.google.prefab.api.NoMatchingLibraryException: No compatible library found for //ReactAndroid/fabricjni. Rejected the following libraries:

> Task :react-native-quick-crypto:configureCMakeDebug[arm64-v8a] FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-quick-crypto:configureCMakeDebug[arm64-v8a]'.
> [CXX1214] <rootDir>\node_modules\react-native-quick-crypto\android\CMakeLists.txt debug|arm64-v8a : User has minSdkVersion 21 but library was built for 23 [//ReactAndroid/fabricjni]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.6/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.        

BUILD FAILED in 1m 23s
44 actionable tasks: 44 executed
```

## Steps to reproduce
1. Run `npx react-native@latest init AwesomeProject`
2. Run `npm i react-native-quick-crypto`
3. Attempt to build for Android.

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "34.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 23
         compileSdkVersion = 34
         targetSdkVersion = 34
         ndkVersion = "25.1.8937393"

